client/src/HomePage.scss

```diff
@@ -44,10 +44,8 @@
 
 .url-div {
   margin-top: 5px;
-  height: 25px;
-  border: none;
-  width: 50vw;
-  border-radius: 5%;
+  display: flex;
+  flex-direction: row;
```

Comment:
This is the default, and so does not need to be written unless if overwritten by a context